src/components/content/header-block/header-block.tsx

```diff
@@ -50,14 +50,14 @@ const Topline = styled.p`
   }
 `;
 
-const Headline = styled.h2<LargeProps>`
+const Headline = styled.h2`
   ${TextStyles.headlineM}
   letter-spacing: -0.01em;
   color: #202840;
-  margin-top: 16px;
-  margin-bottom: ${({ large }) => (large ? '32px' : '16px')};
```

Comment:
There is no place in the design where the margin has to be changed because of `large`. Therefore, this distinction is removed